dashboard/app/api.go

```diff
@@ -801,6 +808,30 @@ func parseCrashAssets(c context.Context, req *dashapi.Crash) ([]Asset, error) {
 	return assets, nil
 }
 
+func detectCrashSubsystems(req *dashapi.Crash, build *Build) []string {
+	extractor := &subsystem.SubsystemExtractor{}
+	nsConfig := config.Namespaces[build.Namespace]
+	if nsConfig.Subsystems != nil {
+		extractor.CallToSubsystems = nsConfig.Subsystems.CallToSubsystems
+	}
+	return extractor.Extract(&subsystem.Crash{
+		OS:          build.OS,
+		GuiltyFiles: req.GuiltyFiles,
+		SyzRepro:    string(req.ReproSyz),
+	})
+}
+
+func subsystemMaintainers(ns, subsystemName string) []string {
+	nsConfig := config.Namespaces[ns]
```

Comment:
If you store Subsystems by value and pre-install empty functions, then this function can be replaced with just `
config.Namespaces[ns].SubsystemCc(subsystemName)`  and detectCrashSubsystems can be simplified:
```
&subsystem.SubsystemExtractor{
   CallToSubsystems: onfig.Namespaces[build.Namespace].Subsystems.CallToSubsystems
}
```